streams/jo.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="" status="online",A One TV
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="AOneTV.jo" status="online",A One TV
```

It is necessary to load then the channel info from the database.

Reply:
Thanks